iron-menu-behavior.html

```diff
@@ -325,8 +334,13 @@
      */
     _onKeydown: function(event) {
       if (!this.keyboardEventMatchesKeys(event, 'up down esc')) {
+        this.cancelDebouncer('_clearSearchText');
+        
         // all other keys focus the menu item starting with that character
-        this._focusWithKeyboardEvent(event);
+        this._searchText += String.fromCharCode(event.keyCode).toLowerCase();
```

Comment:
```javascript
String.fromCharCode(event.keyCode)
```
I was going to ask you to change this, then I realized that it's what we currently use. :/

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode
The web *really* needs to figure out what to do about keyboard events. `keyCode`, `key`, `code`, `charCode`, `keyIdentifier`, `which`...